src/ExpressionEvaluator/Core/Source/ResultProvider/Formatter.cs

```diff
@@ -173,6 +175,35 @@ string IDkmClrFullNameProvider.GetClrExpressionForThis(DkmInspectionContext insp
             return _thisString;
         }
 
+        string IDkmClrFullNameProvider2.GetClrNameForLocalVariable(DkmInspectionContext inspectionContext, DkmClrModuleInstance moduleInstance, DkmClrMethodId methodId, DkmILRange iLRange, DkmClrLocalVariable localVariable)
+        {
+            string metadataName = localVariable.Name;
+            return metadataName.IsCompilerGenerated() ? SimplifyCompilerGeneratedName(metadataName) : metadataName;
```

Comment:
>IsCompilerGenerated [](http://example.com/codeflow?start=32&length=19)

I'd always call SimplifyCompilerGeneratedName. It returns the unchanged name if the metadata name is not compiler generated.